EazFixer.Code/Other/Logger.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.IO;
+
+namespace EazFixer.Code.Other {
+    internal class Logger {
+        private static StreamWriter stream_;
```

Comment:
Why does this have a _ suffix? 😕 Please follow the naming style of the rest of project.